lib/brakeman/tracker/config.rb

```diff
@@ -174,6 +191,20 @@ def lower? lhs, rhs
       end
     end
 
+    # Brakeman does not support Haml 5 yet. See `check_haml_version`.
+    #
+    # This method is messy because all we have is a version `Requirement` (like
+    # `~> 5.0.3`, or `> 4.99`) The only way I could think of was to loop over
+    # known Haml 5 versions and test whether the `Requirement` is satisfied. I
+    # included '5.99.99' in the list so that this method might stand a chance of
+    # working in the future.
+    def supported_haml_version?
```

Comment:
In many cases, there will be an exact gem version from `Gemfile.lock`.

I am okay with just using `version_between?` which will handle most cases.

Reply:
`version_between?` expects a single version number, but all we have is a "requirement", right? Consider the requirement `> 4.99`. This requirement will install Haml 5.  It would be incorrect to ignore the operator and only consider the number, 4.99.  `version_between?` would think it is Haml 4.

Reply:
That is an edge case, yes, but again most applications are going to have a lock file with the exact version, not just the requirement.